ui/app/models/node.js

```diff
@@ -127,9 +127,9 @@ export default class Node extends Model {
     if (this.isDraining || !this.isEligible) {
       return 'alert-circle-fill';
     } else if (this.status === 'down') {
-      return 'cancel-circle-fill';
+      return 'x-circle-fill';
     } else if (this.status === 'initializing') {
-      return 'node-init-circle-fill';
```

Comment:
This is one of the few icons that Helios doesn't have an equivalent for. Showed 2 machines stacked vertically, one greyer than the other.

Reply:
You can asks for a new icon to be added to the HDS library using this form: https://go.hashi.co/hds-support